iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/gcp/AuditLogMemberExclusionCheck.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * SonarQube IaC Plugin
+ * Copyright (C) 2021-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonar.iac.terraform.checks.gcp;
+
+import java.util.function.Predicate;
+import org.sonar.check.Rule;
+import org.sonar.iac.terraform.api.tree.AttributeTree;
+import org.sonar.iac.terraform.api.tree.TupleTree;
+import org.sonar.iac.terraform.checks.AbstractNewResourceCheck;
+
+@Rule(key = "S6414")
+public class AuditLogMemberExclusionCheck extends AbstractNewResourceCheck {
+
+  @Override
+  protected void registerResourceConsumer() {
+    register("google_project_iam_audit_config",
+      resource -> resource.blocks("audit_log_config")
+        .forEach(block -> block.list("exempted_members")
+          .reportIf(isNotEmpty(), "Make sure excluding members activity from audit logs is safe here.")));
+  }
+
+  private static Predicate<AttributeTree> isNotEmpty() {
+    return attribute -> attribute != null && !((TupleTree) attribute.value()).elements().trees().isEmpty();
```

Comment:
The cast to `TupleTree` seems risky here. 
As a general rule, I prefer to cast only right after having tested that the cast will be safe.
It seems that `ListSymbol` already has a check about `Kind.TUPLE` so I would re-use that one: for example, `ListSymbol` may expose a method named `elements()` or `isEmpty()`.
I'm available to talk about that.

Reply:
Having a `elements()` method is fine for me.